zingo-status/src/confirmation_status.rs

```diff
@@ -198,21 +128,26 @@ impl ConfirmationStatus {
     /// use zingo_status::confirmation_status::ConfirmationStatus;
     /// use zcash_primitives::consensus::BlockHeight;
     ///
-    /// let status = ConfirmationStatus::Confirmed(16.into());
-    /// assert_eq!(status.is_pending_before(&15.into()), false);
-    ///
-    /// let status = ConfirmationStatus::Mempool(12.into());
-    /// assert_eq!(status.is_pending_before(&13.into()), true);
-    ///
-    /// let status = ConfirmationStatus::Mempool(14.into());
-    /// assert_eq!(status.is_pending_before(&14.into()), false);
-    /// ```
+    /// assert!(!ConfirmationStatus::Calculated(10.into()).is_pending_before(&9.into()));
+    /// assert!(!ConfirmationStatus::Calculated(10.into()).is_pending_before(&10.into()));
+    /// assert!(!ConfirmationStatus::Calculated(10.into()).is_pending_before(&11.into()));
+    /// assert!(!ConfirmationStatus::Transmitted(10.into()).is_pending_before(&9.into()));
+    /// assert!(!ConfirmationStatus::Transmitted(10.into()).is_pending_before(&10.into()));
+    /// assert!(ConfirmationStatus::Transmitted(10.into()).is_pending_before(&11.into()));
+    /// assert!(!ConfirmationStatus::Mempool(10.into()).is_pending_before(&9.into()));
+    /// assert!(!ConfirmationStatus::Mempool(10.into()).is_pending_before(&10.into()));
+    /// assert!(ConfirmationStatus::Mempool(10.into()).is_pending_before(&11.into()));
+    /// assert!(!ConfirmationStatus::Confirmed(10.into()).is_pending_before(&9.into()));
+    /// assert!(!ConfirmationStatus::Confirmed(10.into()).is_pending_before(&10.into()));
+    /// assert!(!ConfirmationStatus::Confirmed(10.into()).is_pending_before(&11.into()));
+    /// ```
+    // TODO remove 'pending' and fix spend status.
     pub fn is_pending_before(&self, comparison_height: &BlockHeight) -> bool {
         match self {
             Self::Transmitted(self_height) | Self::Mempool(self_height) => {
```

Comment:
why is `Calculated` not in the same category as `Transmitted` and `Mempool` when it comes to confirmation? it strikes me that this could cause a pending bug?